lessons/getting-started-with-mysql-using-r.md

```diff
@@ -0,0 +1,826 @@
+
+{% include figure.html filename="getting-started-with-mysql-9.png" caption="Write down and then type in a root password" %}
+
+###### 6. Windows Service
+Settings here are optional, but I find it easier to set up MySQL as a Windows Service and have it start automatically. A Windows Service is a process that runs on your computer while you are working. You can change your Windows Service settings later to start the MySQL service manually so that it does not start when you don't need it.
+
+{% include figure.html filename="getting-started-with-mysql-10.png" caption="MySQL as a Windows Service" %}
+
+Click the Execute and Next buttons to finish the installation and start the server.  
+
+###### 7. Root password
+When prompted for the root password, enter the root password you wrote down in step *5. Accounts and Roles* above.
+(See below)
+
+{% include figure.html filename="getting-started-with-mysql-15.png" caption="Root password" %}
+
+Look in the Start menu under MySQL for MySQL Workbench.  If it is there, click to launch.  If not, click on MySQL Installer - Community to re-run the installation and add MySQL Workbench to the installed products.
+
+##### Installation tips for a Mac
+
+###### 1. Install MySQL Community Server
+Once the MySQL Community Server install file is downloaded, double click on the downloaded file to install it.  (See below)
+
+{% include figure.html filename="getting-started-with-mysql-17.png" caption="Install file" %}
+
+###### 2. Write down temporary password
+Follow the prompts to accept the licence and installation location.  **Important: There will be a prompt for a temporary password.  Write this down carefully.**  (See below for an example.  Your temporary password will be different than what is shown below.) If you make an error, you can remove the installed server and reinstall it, but that is a minor hassle. One of the lesson reviewers found [this StackOverflow answer](https://stackoverflow.com/a/37524283) helpful for this part.
+
+{% include figure.html filename="getting-started-with-mysql-18.png" caption="Temporary password" %}
+
+Once installed, we are going to change the root password for the MySQL server.
+
+###### 3. Change the root password for the MySQL server
+
+**This section of the lesson has caused difficulty for others.  Take your time and please note that the MySQL commands end with a semicolon. Watch for them in some of the commands below.**
+
+3.1. Open a terminal window
+
+3.2. Add /usr/local/mysql/bin to the PATH per the command below. The PATH is a list of directories the computer looks in when you type a command to run a program.  In the next step below, when you run *mysql*, the PATH searches the directories it has for the *mysql* program.  PATH finds *mysql* in the directory */usr/local/mysql/bin* and runs it.  PATH just saves typing the whole path, in this case  */usr/local/mysql/bin/mysql*, to a program when you want to run it.
```

Comment:
Across this lesson, I've seen a few different kinds of conventions for styling text like file paths and commands that need to be entered into the console. Sometimes they're bare like this, other times _italicized_, other times **bold**.

I usually like to wrap these paths, and any kind of text that needs to go into a code file or get entered into the terminal, with `backticks so that they look like this` however this hasn't been something that we have consistently enforced across our lessons yet. 

Whichever way you go, it'd probably be good to get these strings normalized across the whole lesson text so that it's easier to understand what kind of text is "words that are code or commands that go into the console" versus "words marked bold because they're an important vocab word"